### ISSUE TITLE
Device request Sonoff Mini Wifi 

### ISSUE THREAD

**Issue by konradwalsh** [robertklep/name.klep.sonoff#38]
Hey
Are you able to add support for sonoff mini?

**Comment by robertklep** [comment]
No, I stopped development for this app. However, if you're using Tasmota and configure the device as a Sonoff Basic, it might just work.
